src/components/Main/Main.js

```diff
@@ -135,6 +140,11 @@ export class MainComponent {
       return;
     }
 
+    if (/albums\/\w+/.test(pathname)) {
```

Comment:
you could extract song id here and pass it to component in mount() or in other method

Reply:
Corrected